start.sh

```diff
@@ -82,7 +84,7 @@ done
 for CONTAINER in $CONTAINERS; do
 	URL_VAR="URL_${CONTAINER//-/_}"
 	echo "Starting ${!URL_VAR}"
-	open ${!URL_VAR} 2>/dev/null || x-www-browser ${!URL_VAR}
+	open ${!URL_VAR} 2>/dev/null || x-www-browser ${!URL_VAR} &
```

Comment:
This does not seem to have the intended effect, if your intended effect is to keep the browser in the background when opening. The `-g` argument for `open` also does not seem to work. I would be more in favor of removing this completely (for me personally it's more of a hindrance).

Reply:
My intended effect is that, when starting the server, the output does not start until I close the browser and start it up again.

Reply:
I don't mind removing this completely, as it opens Firefox in my WSL/Linux shell, but I want to work on Chrome in my Windows system.